packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -18,12 +19,17 @@ export default async function processDeployment({
 }: any) {
   const { warn, log } = createOutput({ debug });
   let bar: Progress | null = null;
+  const useHttp2 = semver.satisfies(process.version, '>10');
```

Comment:
```suggestion
  const useHttp2 = semver.satisfies(process.version, '>=10.4.0');
```

The docs mention that fetch-h2 requires Node 10.4.0 or newer https://www.npmjs.com/package/fetch-h2#releases